e2e-tests/playwright/support/pageObjects/global-obj.ts

```diff
@@ -4,7 +4,10 @@ export const waitsObjs = {
 };
 
 export const UIhelperPO = {
-  MuiButtonLabel: 'span[class^="MuiButton-label"]',
+  // MUI v4 buttons are spans with a class starting with MuiButton-label
+  // MUI v5 buttons are HTML buttons with a class containing MuiButton-root
+  MuiButtonLabel:
+    'span[class^="MuiButton-label"],button[class~="MuiButton-root"]',
```

Comment:
fyi: I personally would just use button as selector, but I tried to reduce the risk of regressions with this change.